Documentation/devicetree/bindings/phy/phy-cadence-torrent.txt

```diff
@@ -7,6 +7,7 @@ the Cadence MHDP DisplayPort controller.
 -------------------------------------------------------------------------------
 Required properties (controller (parent) node):
 - compatible	: Should be "cdns,torrent-phy"
+		  Use "cdns,ti,torrent-phy" for TI J7 SoCs
```

Comment:
I think we should instead use ti,j721e-serdes-10g? Having two vendor name in compatible might be confusing. Not sure what Rob Herring would prefer.

Reply:
Okay, name can be decided. I have just taken compatible used in Sierra driver in ti-4.19.y branch as a reference. Please let us know the preferred name, we will use that.